Microsoft.Dynamics365.UIAutomation.Api/Pages/Navigation.cs

```diff
@@ -441,10 +454,18 @@ public BrowserCommandResult<bool> OpenPrivacyStatement(int thinkTime = Constants
 
                 var subItems = subNavElement.FindElements(By.ClassName(Elements.CssClass[Reference.Navigation.SubActionElementClass]));
 
-                foreach (var subItem in subItems)
+                foreach (var subItem in subItems )
                 {
+                    
                     if(!string.IsNullOrEmpty(subItem.Text))
-                    dictionary.Add(subItem.Text.ToLowerString(), subItem);
+                      dictionary.Add(subItem.Text.ToLowerString(), subItem);
+                    else //if the subarea is hidden, it won't have "text" populate. GetAttribute("text") seems to have the right value (at least when tested in Chrome)
+                    {
+                        var text = subItem.GetAttribute("text");
```

Comment:
Is this "text" going to be the same of will be changed to "Go to next page" which is the title to move to the hidden sub area?